core/src/main/java/hudson/AboutJenkins.java

```diff
@@ -38,4 +42,9 @@ public URL getLicensesURL() {
         return AboutJenkins.class.getResource("/META-INF/licenses.xml");
     }
 
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
```

Comment:
Should it be replaced by read-only access in JEP-224?

Reply:
I think you are right, as this view can not be used to modify anything. It can be changed to read-only when #4149 is merged, or as part of it maybe.

Reply:
@timja we need this because the change in `ManagementLink#getRequiredPermission`.

If possible, I think would be better to continue the conversation there so it is easier to follow :smile: 

https://github.com/jenkinsci/jenkins/pull/4483/files/78561c11542e692c71442264b2f94ea7e04a0263#diff-97a16a57b311b64546f011915a4ab1ba